core/src/main/java/bio/terra/pearl/core/service/site/LocalizedSiteContentService.java

```diff
@@ -52,6 +52,7 @@ public LocalizedSiteContent create(LocalizedSiteContent localSite) {
             dao.setLandingPageId(savedSite.getId(), landingPage.getId());
         }
         savedSite.setLandingPage(landingPage);
+        savedSite.setFooterSection(localSite.getFooterSection());
```

Comment:
This was not attached to the SiteContent response when a new version was created (so the site footer would disappear from the editor after you saved)

Reply:
I'm still seeing this behavior -- after I save, the footer disappears

Reply:
For good hygiene, I would add `localSite.setFooterSection(footer);` below line 37 above.  that way the attached object will have all the correct ids from the database, in case we ever decide to use them